.travis.yml

```diff
@@ -4,10 +4,11 @@ otp_release:
   - 19.0
   - 20.0
 elixir:
-  - 1.3
   - 1.4
   - 1.5
 env: MIX_ENV=test
+install:
+  - mix ecto.create
```

Comment:
You need to add `mix local.hex --force` and `mix local.rebar --force` here - that should do it I think.

Reply:
Sounds good.  EOD for me here, so I'll try tomorrow

Reply:
I'll go ahead and merge this and add it, no worries.